views/home.erb

```diff
@@ -1,3 +1,49 @@
-<h1>Welcome, <%=patron.full_name %></h1>
+<h1>Welcome, Samuel Higgins</h1>
 <p>This is the home of your University of Michigan Library account.</p>
 
+<!--
+  Could this data come from back-end?
```

Comment:
@niquerio Nothing super important to do immediately, but I figure long-term group this related navigation together in the backend?

Reply:
```ruby
# Ideas for navigation data
[
  {
    label: "Fines & Fees", # heading
    to: "/fines", # path
    notifications: 2, # used for the badge.
    description: "Views information ...", # the homepage description text.
    featured: true, # to tell if it's on the homepage.
    icon: "money" # unique ID for associated icon
  }
]
```